lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.0.0-rc.2",
+  "packages": [
+    "packages/*"
+  ],
+  "version": "independent"
```

Comment:
This looks good, though I'm not sure why you'd want this repo to be independently versioned